docs/contribute.md

```diff
@@ -0,0 +1,48 @@
+# Contributing to Primer
+
+There are several ways you can contribute to and participate in Primer:
+
+- Participate in discussions
+- Submit an issue
+- Improve the documentation
+- Close issues
+- Propose a new component or UI pattern
+- Questions and support
+
+Please note that all contributions to Primer need to follow our code of conduct.
```

Comment:
A few options here:
1. link to a general CoC 
2. not link, and refer that each project/repo has its own CoC
3. link to all relevant CoCs

I think 2 may be the simplest, but potentially less useful.

Reply:
I think #2 works (or 2 and link the general CoC information on GitHub)